build/templates/conf.py.mako

```diff
@@ -0,0 +1,191 @@
+<%
+config = template_parameters['metadata'].config
+%>\
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# NI Modular Instruments Python API documentation build configuration file, created by
```

Comment:
> NI Modular Instruments Python API documentation build configuration file

Right now we only have configuration for build but I think that's too narrow of a scope for this file. This file will contain configuration for the overall nimi-python project.

Reply:
This file is only for documentation.